tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
-# requirements from setup.py
-django-filer>=1.2.4
-djangocms-attributes-field>=0.4.0
-# other requirements
 djangocms-helper
 tox
 coverage
+isort
+flake8
+# override "pyflakes<2.1" from djangocms-helper
```

Comment:
?? this should not work if it is a normal requirement? why is this not a conflict?

Reply:
in this case its installed via pip install -r tests/requirements.txt, it throws a warning but installs it nonetheless